src/globals/objects/_grid.scss

```diff
@@ -1,53 +1,78 @@
-@mixin govuk-grid {
-  @include govuk-clearfix;
-  margin-right: - ($govuk-gutter-half);
-  margin-left: - ($govuk-gutter-half);
-}
+// A mixin for a grid row
+// Creates a grid row with a standardised margin.
+// Usage:
+// Default:
+// @include @include govuk-grid()
+//
+// Provide a custom grid class:
+// @include govuk-grid("app-grid");
 
-@mixin govuk-grid-item {
-  box-sizing: border-box;
-  padding: 0 $govuk-gutter-half;
+@mixin govuk-grid($class: null) {
+  $class: "govuk-grid-row" !default;
 
-  @include mq($from: tablet) {
-    float: left;
+  .#{$class} {
+    @include govuk-clearfix;
+    margin-right: - ($govuk-gutter-half);
+    margin-left: - ($govuk-gutter-half);
   }
 }
 
-@include govuk-exports("grid") {
+// A mixin for a grid column
+// Creates a cross browser grid column with a standardised gutter between the
+// columns. Widths should be defined as fractions of the full desktop width
+// they want to fill. By default they break to become full width at tablet size
+// but that can be configured to be desktop using the `$full-width` argument.
+//
```

Comment:
Hmmm I think `$full-width` sounds like 100% width which could be confusing as that's what the mobile column is. Maybe `$fraction-width`, `$unit-width`... I don't know, naming is hard.

Reply:
went with $full-width as its the same param as in toolkit. maybe advanced users will be familiar with it. 
it's more like "take proportional width at this breakpoint".
`$full-width-from` ?

Reply:
Hmmm so the column is not "full width" from that point on. On mobile/tablet the column is full width (or 100% width). At this breakpoint the column width becomes a fraction/unit. So the name of the variable should be the opposite of "full-width" but I don't know what. `$fraction-width-from`, `$unit-width-from`?

I might not be explaining this very well 😅

Reply:
Or `$break-from`? I'm okay with either, we'll learn more in user research.